trin-core/src/portalnet/overlay_service.rs

```diff
@@ -934,20 +935,21 @@ where
                 ))
             })?;
 
-            requested_keys
-                .set(
-                    i,
-                    self.storage.read().should_store(&key).map_err(|err| {
-                        OverlayRequestError::AcceptError(format!(
-                            "Unable to check content availability: {err}"
-                        ))
-                    })?,
-                )
+            // Accept content if within radius and not already present in the data store.
+            let accept = self
+                .store
+                .read()
+                .is_key_within_radius_and_unavailable(&key)
                 .map_err(|err| {
                     OverlayRequestError::AcceptError(format!(
-                        "Unable to set requested keys bits: {err:?}"
+                        "unable to check content availability {err}"
```

Comment:
*nitpick* but in general, I think it's better overall if we consistently begin error messages with capital letters throughout the codebase